Zend/tests/invalid_parent_const_ref_leak.phpt

```diff
@@ -15,4 +15,4 @@ try {
 
 ?>
 --EXPECT--
-Cannot access parent:: when current class scope has no parent
+Cannot access parent when current class scope has no parent
```

Comment:
I think the usage of `::` is a bit weird, thus the removal. Can the new version still convey the message, right?